instance/models/mixins/openedx_theme.py

```diff
@@ -55,29 +55,40 @@ class OpenEdXThemeMixin(models.Model):
     class Meta:
```

Comment:
Remember to update the frontend API client after implementing the changes by going to `frontend/` and running `./scripts/update-api-client.sh`. The frontend should keep working after these changes are merged.

Reply:
@giovannicimolin, I have not made any changes to the model at all. So will this still be required?

Reply:
@lgp171188  No need, I just tested and checked that the API client didn't break with these new changes.